### PR TITLE
Added "Purge" to CDNsites

### PR DESCRIPTION
**Pull request by ostcrom**
Implemented "Purge" endpoint to CDN Sites class.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
.DS_Store file should not be included. Can you remove it pls?

**Comment by ostcrom** [comment]
Thank you for the guidance. Please let me know if the changes I have just pushed are acceptable.

**Review by ghost — APPROVED** [review]

File: pystackpath/cdnsites.py [hunk 1]
```diff
@@ -34,4 +34,26 @@ def create(self, domain, origin_hostname):
...
+        return self
+
+    def purge(self, url, recursive = True, invalidateOnly = False, purgeAllDynamic = False, headers = [], purgeSelector = []):
+        purgeSelectors = ["selectorType","selectorName","selectorValue", "selectorValueDelimter"]
+
+        for value in purgeSelector:
+            if not value in purgeSelectors:
+                raise ValueError(f"{value} is not a valid purgeSelector: {purgeSelectors}")
+
+        data = {"items": [
```
Comment (prometherion) [inline pystackpath/cdnsites.py]: Please, indent correctly dictionary following [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/#indentation):
```
data = {
    "items": [
        {
            "url": url,
            "recursive": recursive,
            "headers": headers,
            "invalidateOnly": invalidateOnly,
            "purgeSelector": purgeSelector
        }
    ]
}
```

**Comment by sandromodarelli** [comment]
thank you @ostcrom. please, add a row on README about CDN cache purge.
wdyt about add the GetPurgeStatus, too? (maybe with a new PR) https://developer.stackpath.com/en/api/cdn/#operation/GetPurgeStatus

**Comment by ostcrom** [comment]
Thank you for the feedback, I have made the changes requested. Please let me know if any further revision is needed.

**Review by prometherion — APPROVED** [review]
LGTM

**Review by sandromodarelli — APPROVED** [review]

**Pull request merged**